cors/handler.go

```diff
@@ -0,0 +1,44 @@
+package cors
+
+import (
+	"net/http"
+
+	middleware "github.com/SKF/go-enlight-middleware"
+)
+
+type Middleware struct {
+	Tracer middleware.Tracer
+}
+
+// New returns a new cors middleware which adds cors headers to the responses.
```

Comment:
As I see it this middleware does two things,
* Add Access-Control-Allow-Origin on all requests
* Act as the default route for all OPTIONS requests

The naming of this middleware does not really help the user, so a better description is probably a good idea.